dask/dask.yaml

```diff
@@ -7,12 +7,16 @@ tokenize:
   ensure-deterministic: false  # If true, tokenize will error instead of falling back to uuids
 
 dataframe:
+  backend:
+    library: null  # Backend dataframe library for input IO and data creation
   shuffle-compression: null  # compression for on disk-shuffling. Partd supports ZLib, BZ2, SNAPPY, BLOSC
   parquet:
     metadata-task-size-local: 512  # Number of files per local metadata-processing task
     metadata-task-size-remote: 16  # Number of files per remote metadata-processing task
 
 array:
+  backend:
+    library: null  # Backend array library for input IO and data creation
```

Comment:
I think it is important that we be sure to agree on the config structure/label here. We can change many backend-dispatching implementation details after the fact. However, I have a feeling that whatever config name we choose will stick quickly.

Reply:
The original design of this PR involved configurable fall-back behavior that is now the responsibility of the backend entrypoint. At this point, I am leaning in the direction of removing the nesting, and just using `dataframe.backend`.

Reply:
**Update**: I decide to remove the ".library" for now, because I can't really think of a (clean) way that we would want to add/use other standard "array/dataframe.backend.*" config options to be supported for all backend options. Each entrypoint is free to use the dask config system to expose whatever custom options it chooses to implement/support anyway,